llvm/lib/Target/SPIRV/SPIRVCommandLine.cpp

```diff
@@ -163,7 +163,9 @@ static const std::map<std::string, SPIRV::Extension::Extension, std::less<>>
         {"SPV_INTEL_kernel_attributes",
          SPIRV::Extension::Extension::SPV_INTEL_kernel_attributes},
         {"SPV_ALTERA_blocking_pipes",
-         SPIRV::Extension::Extension::SPV_ALTERA_blocking_pipes}};
+         SPIRV::Extension::Extension::SPV_ALTERA_blocking_pipes},
+        {"SPV_EXT_image_raw10_raw12",
+         SPIRV::Extension::Extension::SPV_EXT_image_raw10_raw12}};
```

Comment:
The change LGTM. However, please also add the SPV_EXT_image_raw10_raw12 to SPIRVUsage and rebase the PR since it is not possible to merge after the previous PR adding another extension was merged.